routes/validate.js

```diff
@@ -1,14 +1,17 @@
+var crypto =require('crypto');
 module.exports = function(req, res, next) {
+	var username = req.body.user_name;
```

Comment:
u need to validate using route name. 
e.g if route is /user/login then it wont validate.

not using body